src/ngraph/op/util/attr_types.hpp

```diff
@@ -42,15 +42,17 @@ namespace ngraph
         ///              Floor(num_dims/2) at the beginning and
         ///              Ceil(num_dims/2) at the end
         /// VALID      - No padding
+        /// INVALID    - Used for validation of provided auto padding attribute value.
         ///
         enum class PadType
         {
             EXPLICIT = 0,
             SAME_LOWER,
             SAME_UPPER,
             VALID,
+            INVALID,
```

Comment:
AFAICT this is only used to signal failure of `get_ng_pad_type`. I would prefer a different mechanism for this.